source/components/tooltip/HxTooltip.less

```diff
@@ -0,0 +1,24 @@
+@import 'HxElement';
+@import 'mix-position';
+
+:host {
+    background-color: @gray-0;
+    border-color: @gray-300;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0px 3px 3px 0 @shadow-color;
+    display: none;
+    left: 0;
+    max-width: 25rem; //400px
+    position: absolute;
+    top: 0;
+    z-index: @tooltip-z-index;
```

Comment:
You may want to specify more default styles, because the tooltip element will inherit them from the parent otherwise.

For example, if the parent element `<div class="demo">` has the style `font-size: 2rem`:
![screen shot 2017-12-12 at 1 24 05 pm](https://user-images.githubusercontent.com/6867918/33904037-eedd6412-df3f-11e7-8f0d-633600d25ba2.png)

I've identified at least these for popovers:
```
  color: initial;
  font-family: "Roboto", sans-serif;
  font-size: 1rem;
  font-style: normal;
  font-weight: normal;
  letter-spacing: normal;
  text-align: left;
  text-decoration: none;
  text-shadow: none;
  text-transform: none;
  white-space: normal;
  word-break: normal;
  word-spacing: normal;
  word-wrap: normal;
```

Reply:
This may be the doings of `HxElement`. If you don't want inheritance, just replace `@import 'HxElement';` with `@import 'vars';`